docs/user-docs/latest/content-types-builder/configuring-fields-content-type.md

```diff
@@ -302,6 +302,10 @@ The Enumeration field allows to configure a list of values displayed in a drop-d
 
 ::::
 
+::: danger
+Enumeration values on Strapi v4.1.3 and above should not start with a special character or a number. This would cause a GraphQL issue leading to a server crash and some possible data loss, and will be fixed as soon as possible.
```

Comment:
```suggestion
Since Strapi v4.1.3, enumeration values should always have an alphabetical character preceding any number as it could otherwise cause the server to crash without notice when the GraphQL plugin is installed.
```

Reply:
Thanks, @Convly! That's a good suggestion, though I feel like we might be missing information about special characters 🤔 

If I literally apply the information I understand from the new wording:
- `a4` should work
- should `a_` work?
- should `_a4` work? (we have an "alphabetical character preceding any number" but the enum value still _starts_ with something else)

Just asking to better understand. If you have a list of accepted and non-accepted values (or maybe just a link to the regex that validates it), it would be helpful. Thank you!

Reply:
The key point here is that we're making the checks on the regressed enum values and not on the values entered directly.
Basically, it means we're removing the non alpha-numerical characters and then checking if it starts with a number or not.

- a4 is valid,
- 4a is not valid
- _a4 is valid
- _4a is not valid
- a4_ is valid

Reply:
Thanks for the detailed explanations 🙂 I'm perfectly aligned with your suggestion, then.